vscode-trace-common/package.json

```diff
@@ -11,7 +11,8 @@
     "dependencies": {
         "json-bigint": "sidorares/json-bigint#2c0a5f896d7888e68e5f4ae3c7ea5cd42fd54473",
         "traceviewer-base": "^0.6.0",
-        "tsp-typescript-client": "^0.5.1"
+        "vscode-messenger": "^0.5",
```

Comment:
For consistency, we may want to explicitly use version range "^0.5.0"
(same for line below)